### PR TITLE
typos fixed

### PR DESCRIPTION
**Pull request by yuliaUU**

### CODE REVIEW

**Comment by yuliaUU** [comment]
I only could not sort the issue with  question "What do you have to do if you are working with the terminal and you want to write a different text for this merge conflict?" part 10 in https://toolbox-learn.mds.ubc.ca/en/module4

it says **this merge conflict** - I don't know whether an image is missing or smth else?

**Review by joelostblom — APPROVED** [review]
Great, thank you!

**Comment by joelostblom** [comment]
@flor14 Do you remember which image is meant to show up in the question Yulia is referring too? It looks like this now:

image

**Comment by flor14** [comment]
Probably one of a merge conflict in VS Code. I would check the image folder
to see if it is there.

**Comment by joelostblom** [comment]
Yes, I had a look there already but there are quite a few and the names don't give a away which one it is, so that is why I wondered if you remembered.

image

But after going through all the images, it seems like it could be [vc-merge-conflict-t.png](https://github.com/UBC-MDS/ds-toolbox/blob/main/static/module4/vc-merge-conflict-t.png), @yuliaUU does that make sense for you? Maybe we can add a response option for "Compare changes" since there is an arrow pointing at this in the picture?

When I grep the repo it seems like this file is not used anywhere else, so I think it is safe to use without risking duplication:

image

At some point later maybe we can clean out some of the other merge conflict images, since it seems like only a few of them are actually referenced in a chapter? But that's not high priority.

**Comment by flor14** [comment]
Yes, the problem with this section was that there was an update on how
VSCode handles merge conflicts so I have to make a last-minute update
before recording.
I never had time to fix the images and remove the old ones.

**Comment by joelostblom** [comment]
Ah ok that makes sense, thank you!

**Comment by yuliaUU** [comment]
the only problem with that image:  [vc-merge-conflict-t.png](https://github.com/UBC-MDS/ds-toolbox/blob/main/static/module4/vc-merge-conflict-t.png), is that we don't teach resolving conflict in VSCode

**Comment by joelostblom** [comment]
Hmm ok, I thought we had something about VS code for merge conflicts, but maybe we changed that too. If we don't, then that makes sense, can you make a PR with that image? Maybe we need to reword the question too?

**Comment by yuliaUU** [comment]
ok! I will ty to figure out how to add image ! and i will rewrite a question and to be more relevant to the course

**Comment by yuliaUU** [comment]
@joelostblom please check the new version of this question 

I think it is not possible to add images in these quizzes (at least no modules quizzes has images), so I made it slightly different.

File: chapters/en/module4.md [hunk 1]
```diff
@@ -196,14 +196,25 @@ Is the third options really correct?
...
+  
 <choice id="1">
-<opt text='Accept current change' >
+<opt text='You should delete everything between <code> <<<<<<< </code> and <code> ======= </code> and leave the remaining text' >
 </opt>
-<opt text='Accept incoming change'>
+<opt text='You should delete everything between <code> ======= </code> and <code> >>>>>>> </code>  and leave the remaining text'>
 </opt>
-<opt text='Accept both changes'>
+<opt text='You should delete everything between <code> <<<<<<< </code> and <code> >>>>>>> </code> but keep all the text'>
```
Comment (joelostblom) [inline chapters/en/module4.md:217]: Hmm I don't understand what it means to delete everything AND keep all the text? Maybe we can just remove this alternative? I'm ok with you merging after removing this one @yuliaUU

Comment (yuliaUU) [reply chapters/en/module4.md:217]: yep! remove it!  i meant to remove all >>>> , <<< and === keeping the text

**Pull request merged**